generator/svelte.config.js

```diff
@@ -27,7 +27,7 @@ const config = {
         "img-src": ["'self'", "data:"],
         "style-src": ["'self'", "'unsafe-inline'"],
         "script-src": ["'self'"],
-        "connect-src": ["'self'", "api.tags.ioxio.dev", "api.tags.ioxio.io"],
+        // "connect-src": ["'self'", "api.tags.ioxio.dev", "api.tags.ioxio.io"],
```

Comment:
You don't just remove things like this just because it causes issues, you fix them